### PR TITLE
Remove browser notifications feature

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?
Removing the liveblog notifications 💀 

Even though this is a very interesting feature, it has been broken in production for more than 2 months without anybody (readers or internal) noticing it. 
Given there is no plan to spend time (and love) on this feature, it has been decided to remove it, making the codebase less complicated and save on servers that are propelling this feature.
Please contact @stephanfowler if you feel strongly against this decision

@guardian/dotcom-platform Can you make sure I have not been over-zealous in my deleting? Thank you

## What is the value of this and can you measure success?
Less code, less ec2 instances

## Does this affect other platforms - Amp, Apps, etc?
Only liveblogs

## Tested in CODE?
I have tested that CODE doesn't explode 💥  but the feature was already not working in CODE

### CODE REVIEW

**Review by SiAdcock — APPROVED** [review]
Looks good from a client-side perspective 👍

**Comment by gtrufitt** [comment]
💔

File: applications/app/templates/serviceWorker.scala.js [hunk 1]
```diff
@@ -92,63 +92,3 @@ this.addEventListener('fetch', function (event) {
         event.respondWith(forbidden);
     }
 });
-
-self.addEventListener('push', function (event) {
```
Comment (rich-nguyen) [inline applications/app/templates/serviceWorker.scala.js]: I do wonder what useful functionality we want to keep in our service worker. I can see the IAS exclusion, which we may not need anymore, and the credentials work around, which I'm also not sure we need anymore.

**Comment by janua** [comment]
😢

File: applications/app/templates/serviceWorker.scala.js (hunk 1 above)
Comment (rich-nguyen) [inline applications/app/templates/serviceWorker.scala.js]: (The IAS exclusion never worked on iframes anyway)

**Review by rich-nguyen — APPROVED** [review]

File: applications/app/templates/serviceWorker.scala.js (hunk 1 above)
Comment (gustavpursche) [inline applications/app/templates/serviceWorker.scala.js]: The assets cache is a quite important and should remain there ...

**Comment by DiegoVazquezNanini** [comment]
:+1: 
LGTM Thanks for doing this @TBonnin

File: applications/app/templates/serviceWorker.scala.js (hunk 1 above)
Comment (rich-nguyen) [inline applications/app/templates/serviceWorker.scala.js]: I spotted it the cache usage, but not sure I understand why we need it. we don't explicitly push, so this appears to behave like traditional caching? Thought that the cache will still  be consulted anyway. If it isn't accessed by default, it's odd for the cache spec to implicitly `push`, but not implicitly `match`.

**Pull request merged**